environment/src/main/resources/schema/app/20220509121640_CB-16771_add_platform_column.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-16771 introduce platform to differentiate PaaS and hybrid setup (PaaS Env + SaaS SDX)
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE environment ADD COLUMN IF NOT EXISTS platform VARCHAR(255);
```

Comment:
I don't have any better but for me platform is overloaded with cloud platform. 
We should have something unique that easy to distinguish.
deploymentType, underPinning, bedrock, foundation